settings/Eventbrite.setting.php

```diff
@@ -55,8 +55,8 @@
     'description' => E::ts('Eventbrite system ID for the Eventbrite organization to which all linked events will belong.'),
     'title' => E::ts('Eventbrite API Organization ID'),
     'html_type' => 'Text',
-    // Omitting this line causes the setting to be omitted from the Settings form:
-    // 'quick_form_type' => 'Element',
+    // adding it back to implement the new api.
+    'quick_form_type' => 'Element',
```

Comment:
Hi @harshal6 This has the effect of making the 'Eventbrite API Organization ID' setting editable by the site admin, but I'm not sure this is advisable. The extension operates under the assumption that the given 'Eventbrite Personal OAuth Token' is only in use for one organization, and it then sets the 'Eventbrite API Organization ID' setting automatically. Have you found some need to edit this value manually?

Reply:
yes no special need, initially I did not realise we had organization which was autopopulated so made it editable, let me update the code and make it non editable. @twomice thanks

Reply:
There is no specific need, initially I did not realise we had organization which was autopopulated so made it editable, let me update the code and make it non editable. @twomice thanks